package-templates/react/.mocharc.js

```diff
@@ -1 +1 @@
-module.exports = require("../../presets/mocha.config.js");
+module.exports = require("../../mocha.config.js");
```

Comment:
```suggestion
module.exports = require('../../mocha.config.js');
```

i think this file shouldn't be needed, any reason to have it?

Reply:
No, you are right. Should be removed. Thought mocha doesn't have config lookup like babel does but after renaming mocha.config.js to .mocharc.js it worked